02_bag.ipynb

```diff
@@ -455,8 +457,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "is_even = lambda x: x % 2\n",
-    "b.foldby(is_even, binop=max, combine=max).compute()"
+    "b.foldby(lambda x: x % 2, binop=max, combine=max).compute()"
```

Comment:
With the way the lambda is currently written, the function name really should be `is_odd`. I decided not to rename and function and instead inline the lambda in order to keep consistency with the previous cells.

![image](https://user-images.githubusercontent.com/17503044/100467740-0237a080-30a1-11eb-88af-0283be1345c4.png)